content/04-guides/02-deployment/00-deploying-to-heroku.md

```diff
@@ -0,0 +1,283 @@
+---
+title: 'Deploying to Heroku'
+metaTitle: 'Deploying to Heroku'
+metaDescription: 'Learn how to deploy a Node.js server that uses Prisma to Heroku.'
+---
+
+## Overview
+
+In this guide, you will set up and deploy a Node.js server that uses Prisma with PostgreSQL to [Heroku](https://www.heroku.com). The application will expose a REST API and use Prisma Client to handle fetching, creating, and deleting records from a database.
```

Comment:
It might be worth pointing out that REST is just used as an example and that deploying a GraphQL API works pretty much in the same way